datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/role/BatchAssignRoleResolver.java

```diff
@@ -37,19 +37,8 @@ public CompletableFuture<Boolean> get(DataFetchingEnvironment environment) throw
 
     return CompletableFuture.supplyAsync(() -> {
       try {
-        Urn roleUrn = Urn.createFromString(roleUrnStr);
-        if (!_roleService.exists(roleUrn, authentication)) {
-          throw new RuntimeException(String.format("Role %s does not exist", roleUrnStr));
-        }
```

Comment:
I think we could still keep this check if `roleUrn != null` right? just in case someone sends a garbage role that isn't `null`